miscellaneous/gene_categorizations.py

```diff
@@ -9,6 +9,7 @@
 import sys
```

Comment:
There seems to be a lot of duplication between this file and `summarize_gene_citations_all_species.py`.  For maintainability, having only one copy would be best -- e.g. one of the Python files could import functions from the the other.

I don't think such refinement is necessary for this PR, but it'd be helpful to note it as a TODO comment in both this file and `summarize_gene_citations_all_species.py`.

Reply:
I agree!